.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+go_import_path: rsc.io/grind
+go:
+  - 1.4.3
+  - 1.5.3
+  - 1.6.2
+  - 1.7.1
+  - tip
```

Comment:
Are you sure you want tip to not be an allowed failure? It may cause CI to have false positive failures if tip has temporary issues.

I usually make tip a fast-finish build with allowed failure. E.g., see [here](https://github.com/shurcooL/httpfs/blob/9b25e213bae5873c803ae9b020debada75cf5461/.travis.yml#L6-L9).

Just sharing this in case it's helpful.